examples/generalized_linear_models/GLM.ipynb

```diff
@@ -24,7 +24,7 @@
     "import os\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    print(f"Running on PyMC3 v{pm.__version__}")
</pre><p>While you are here, could you replace all instances of <code>pymc3</code> with <code>pymc</code> and ensure that it runs?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/317/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@fonnesbeck @aloctavodia Is there a version/branch of `bambi` that is running on/with `pymc`? I don't see anything on github to try

Reply:
There is not yet, there is a v4 PR open but its not yet working https://github.com/bambinos/bambi/pull/496

Reply:
Okay, so I won't make the other changes @fonnesbeck mentioned just yet.